test/e2e/specs/onboarding/ftme__sell.ts

```diff
@@ -164,10 +164,14 @@ describe( DataHelper.createSuiteTitle( 'FTME: Sell' ), function () {
 			startSiteFlow = new StartSiteFlow( page );
 		} );
 
-		it( 'Continue with simple option', async function () {
-			await page.waitForURL( /.*setup\/storeFeatures.*/ );
-			await startSiteFlow.clickButton( 'Continue' );
-		} );
+		// This step is only applicable if the flag themes/plugin-bundling
+		// is false.
+		// When the flag is turned on for all environments, delete this.
+		// -mreishus 2022-09-13
+		// it( 'Continue with simple option', async function () {
```

Comment:
**suggestion:optional**: `it.skip` would also work here, with the bonus that it would report the skippage of the step to TeamCity. 

But as stated, this is optional.

Reply:
Seems like a good idea!